packages/pwa-kit-react-sdk/src/ssr/server/react-rendering.js

```diff
@@ -308,6 +318,26 @@ const renderApp = (args) => {
     if (error && isRemote()) {
         delete error.stack
     }
+    performance.mark(PERFORMANCE_MARKS.totalEnd)
+
+    let performanceMetrics = []
+    try {
+        performanceMetrics = getPerformanceMetrics()
+
+        // DISCUSSION! HOW DO WE WANT TO CONTROL THIS?
+        const TBD_FEATURE_FLAG = true
+        if (TBD_FEATURE_FLAG) {
+            console.debug('======Performance metrics======')
+            performanceMetrics.forEach((metric) => {
+                console.debug(`${metric.name} - ${metric.duration}`)
+            })
+            console.debug('===============================')
+        }
+    } catch (e) {
+        console.warn('Failed to get performance metrics', e)
+    }
+    performance.clearMarks()
```

Comment:
In case some other libraries/code have also added performance markers, let's not clear all markers. We can specify the name of the markers we want to clear.

Reply:
fixed